tox.ini

```diff
@@ -7,8 +7,8 @@ envlist =
 requires =
     setuptools >=56, !=61.0.0
     pip >= 19.3.1
-    tox-pypi-filter >= 0.12
-isolated_build = true
+    # Currently tox-pypi-filter is not compatible with tox 4
+    #tox-pypi-filter >= 0.12
```

Comment:
Do we actually use `tox-pypi-filter` for anything? Looks like it was added to the config by @nabobalis in https://github.com/sunpy/sunpy/pull/5161, but that PR didn't actually invoke it using `--pypi-filter`, and it doesn't look like we currently invoke it.

Reply:
It's enabled automatically by it being in the config.

Reply:
🤔 but the [tox-pypi-filter readme](https://github.com/astrofrog/tox-pypi-filter#readme) doesn't say anything about it enabling anything automatically

Reply:
(not disagreeing, with you, just trying to work out what it's doing and why we can't just stop using it!)

Reply:
tox pypi filter allows us to globally exclude versions of dependencies, or even transitive dependencies which are causing the tests to fail without pinning them down in our package metadata. For example, if lxml which is a dependency of zeep had an issue we can ignore it until a bugfix is released for all our tox tests. Or if there's a specific issue with pytest itself we can ignore it for all packages using our pins file in one commit.